Libs/Optimize/Optimize.cpp

```diff
@@ -627,20 +633,36 @@ void Optimize::Initialize()
   m_sampler->GetLinkingFunction()->SetRelativeGradientScaling(m_initial_relative_weighting);
   m_sampler->GetLinkingFunction()->SetRelativeEnergyScaling(m_initial_relative_weighting);
 
+  // Debuggg
+  //std::cout << "Before adding single point" << std::endl;
+  //m_sampler->GetParticleSystem()->PrintParticleSystem();
+

```

Comment:
Can we remove these commented out debug sections?

Reply:
I've marked all these comments with // debuggg. I will still be using them in the near future, so I'd like to keep them for now. I'll remove them once I have no use for them.